pkg/model/uimodel.go

```diff
@@ -53,8 +53,8 @@ func NewUIModel(extraLabels []string) *UIModel {
 	pager.ActiveDot = activeDot
 	pager.InactiveDot = inactiveDot
 	return &UIModel{
-		// red to green
-		progress:    progress.New(progress.WithGradient("#ff0000", "#04B575")),
```

Comment:
Can you add a flag for swapping the colors (e.g. `--invert-colorbar`?), with the current coloring being the default?  I picked red to green intentionally as low utilization isn't the goal and thought it made more sense to show red in that case.

Reply:
Yup, will delete this PR and create a new one - added this flag just now

Reply:
Sorry, shouldn't have done that - apologies. Let me know if this works.